app/views/workspaces/_workspace.html.erb

```diff
@@ -1,7 +1,18 @@
-<li>
-  <%= workspace.name %>
-  <% if in_edit_mode %>
-    <%= link_to 'Show', workspace %>
-    <%= link_to 'Destroy', workspace, method: :delete, data: { confirm: 'Are you sure?' } %>
-  <% end %>
-</li>
+<div class="col-4">
+  <div class="grid-item">
+    <h3><%= link_to workspace.name, workspace %></h3>
```

Comment:
Yeah, that did come up, although we cut bait on our original truncation strategy and went with rails truncation until we could come up w/ a more holistic approach (which we could use for titles and descriptions).

Would you be 🆗 w/ creating a ticket for that as something to follow up on immediately after shipping this?